lms/views/admin/application_instance.py

```diff
@@ -264,9 +264,13 @@ def show_instance(self):
     def update_instance(self):
         ai = self._get_ai_or_404(**self.request.matchdict)
 
-        for ai_field in ["lms_url", "deployment_id"]:
```

Comment:
Keeping this while having `update_application_instance` was a bit odd. Doing it all in one place now.